<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthPrompt</class>
 <widget class="QDialog" name="AuthPrompt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Demande d'autorisation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QPushButton" name="accept_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">Login</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="login_label">
       <property name="text">
        <string notr="true">Login:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pass_label">
       <property name="text">
        <string notr="true">Password:</string>
       </property>
       <property name="buddy">
        <cstring>pass_input</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string notr="true">Annuler</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="Input" name="pass_input">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QComboBox" name="login_input"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Input</class>
   <extends>QLineEdit</extends>
   <header>gui/input_widget</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>login_input</tabstop>
  <tabstop>pass_input</tabstop>
  <tabstop>accept_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>accept_button</sender>
   <signal>clicked()</signal>
   <receiver>AuthPrompt</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>AuthPrompt</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
